### ISSUE TITLE
Move Examples PRs from main repo

### ISSUE THREAD

**Issue by fpliger** [pyscript/pyscript-collective#6]
There are many PRs on the main repo that are stale, waiting for a home. Would be amazing to be able to move them here asap :)

**Comment by marimeireles** [comment]
I have an incomplete list for that!

[#278](https://github.com/pyscript/pyscript/pull/278)  
[#158](https://github.com/pyscript/pyscript/pull/158)  
[#113](https://github.com/pyscript/pyscript/pull/113)  
[#141](https://github.com/pyscript/pyscript/pull/141)  
[#40](https://github.com/pyscript/pyscript/pull/40)  
[#39](https://github.com/pyscript/pyscript/pull/39)  
[#398](https://github.com/pyscript/pyscript/pull/398)  
[#151](https://github.com/pyscript/pyscript/pull/151)  
[#512](https://github.com/pyscript/pyscript/pull/512#issuecomment-1160644326)

But we also have some stuff: https://community.anaconda.cloud/c/tech-topics
Like https://community.anaconda.cloud/t/pyscript-chess-match-viewer-demo/22305

And of course... All over the internet!

**Comment by marimeireles** [comment]
Move example: https://github.com/pyscript/pyscript/pull/686#issuecomment-1246079643

**Comment by marimeireles** [comment]
@pauleveritt not sure where we were having our conversation about moving examples/reaching out to people with examples.
But this issue might be slightly more updated than the comment I sent you.
Cheers.

**Comment by pauleveritt** [comment]
Yep, once we get the new PyScript person looped in, we can start processing these examples.
